modules/rules_cue/metadata.json

```diff
@@ -18,7 +18,8 @@
         "0.4.0",
         "0.4.1",
         "0.4.2",
-        "0.5.0"
+        "0.5.0",
+        "0.7.0"
```

Comment:
I hadn't noticed previously that version 0.6.0 never made it into the BCR.

Reply:
Mentioning that missing version here isn't enough; we also need the files in the _../0.6.0_ directory to be populated accordingly.